_search-plugins/sql/limitation.md

```diff
@@ -77,7 +77,9 @@ The query with `aggregation` and `join` does not support pagination for now.
 
 ## Query processing engines
 
-The SQL plugin has two query processing engines, `V1` and `V2`. Most of the features are supported by both engines, but only the new engine is actively being developed. A query that is first executed on the `V2` engine falls back to the `V1` engine in case of failure. If a query is supported in `V2` but not included in `V1`, the query will fail with an error response.
+Before 3.0.0, the SQL plugin has two query processing engines, `V1` and `V2`. Most of the features are supported by both engines, but only the new engine is actively being developed. A query that is first executed on the `V2` engine falls back to the `V1` engine in case of failure. If a query is supported in `V2` but not included in `V1`, the query will fail with an error response.
+
+From 3.0.0, the SQL plugin introduces a new query engine (`V3`) which leverages Apache Calcite to optimize and execute query. As an experimental feature in v3.0.0, the `V3` engine disabled by default. To enable this new engine, please set `plugins.calcite.enabled` to true. Similar, a query that is first executed on the `V3` engine falls back to the `V2` engine in case of failure. Read [PPL Engine V3](https://github.com/opensearch-project/sql/blob/main/docs/dev/intro-v3-engine.md) for details.
```

Comment:
the `V3` engine is disabled by default

Reply:
Similar to the 'V2' -> 'V1' logic, a query that....

Reply:
done